test-cases/RMLTC0010c-XML/mapping.ttl

```diff
@@ -20,5 +20,5 @@
 
   rr:predicateObjectMap [
     rr:predicate ex:code ;
-    rr:objectMap [ rr:template "\{\{\{ {ISO3166} \}\}\}"; rr:termType rr:Literal]
+    rr:objectMap [ rr:template "\\{\\{\\{ {\"ISO 3166\"} \\}\\}\\}"; rr:termType rr:Literal; ]
```

Comment:
Indeed, here there should be no quotation marks, as an XML tagname can always be retrieved without quotations (bc [an XML tagname can't have spaces](https://www.w3.org/TR/xml/#sec-common-syn)). It should be 

```turtle
     rr:objectMap [ rr:template "\\{\\{\\{ {ISO3166} \\}\\}\\}"; rr:termType rr:Literal; ] 
```